code/pizzabot.js

```diff
@@ -4,4 +4,73 @@ const pepperoni = "Pepperoni Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+// validateOrderName() which should take the orderName variable as an argument and return true or false if the pizza is on the menu or not.
+// const validateOrderName = (orderName) => {
+//   if ((orderName === hawaiian) || (orderName === vegetarian) || (orderName === pepperoni)) {
+//     return true;
+//   } else {
+//     return false;
+//   }
+// }
+
+// calculateTotalCost() which takes orderQuantity and pizzaPrice as arguments and returns the total cost for the order.
+const calculationTotalCost = (orderQuantity, pizzaPrice) => {
+  return orderQuantity * pizzaPrice;
+}
```

Comment:
Very nice :) Another tip from my side - since we have only one line inside `calculationTotalCost` function, we can write it in "arrow function shorthand" :

`const calculationTotalCost = (orderQuantity, pizzaPrice) => orderQty * pizzaPrice;`

We can do that because in arrow functions we can omit curly brackets (`{}`) and `return` keyword if we only have one line of code inside.
For further reference on that subject, please check out https://www.w3schools.com/js/js_arrow_function.asp